guides/source/upgrading_ruby_on_rails.md

```diff
@@ -834,6 +834,19 @@ resp.content_type #=> "text/csv; header=present; charset=utf-16"
 resp.media_type   #=> "text/csv"
 ```
 
+### New `config.hosts` setting
+
+Rails now has a new `config.hosts` setting for security purposes. This setting defaults to `localhost` in development. If you use other domains in development you need to allow them like this:
```

Comment:
Small nitpick. Can you reduce the line lenght to 80-100 characters?
```suggestion
Rails now has a new `config.hosts` setting for security purposes. This setting defaults to
`localhost` in development. If you use other domains in development you need to allow
them like this:
```

Reply:
Agreed. I just kept without line breaks because other paragraphs were missing them as well, so I thought this was the convention.

Reply:
Sometimes it's requested for guides: https://github.com/rails/rails/pull/37885#discussion_r400931085
But it's not really explicitly [documented](https://github.com/rails/rails/pull/42836) yet.